acceptance/bundle/deploy/python-notebook/script

```diff
@@ -0,0 +1,10 @@
+cd my_project
+
+envsubst < databricks.yml.tmpl > databricks.yml
+
+trace $CLI bundle deploy
+
+# FIXME: doesn't work because fake state is shared and tests run in parallel
```

Comment:
Please use Badness field in test.toml (git grep for examples).

Reply:
I think we should just fix the fake. It isn't hard